com/course/zenkouv/src/main/java/lecture6/gitHub/GitHub.java

```diff
@@ -0,0 +1,27 @@
+package lecture6.gitHub;
+
+import java.util.Map;
+
+import lecture6.constant.GitHubApiConstants;
+import lecture6.constant.RequestType;
+import lecture6.executors.IRequestExecutor;
+import lecture6.gitHub.controller.GitHubController;
+
+public class GitHub {
+
+    private GitHubController gitHubController;
+
+    public GitHub(IRequestExecutor executor) {
+        gitHubController = new GitHubController(executor);
+    }
+
+    public void sendCollaboratorInvite(Map<String, String> params, String name) {
+        gitHubController.request(GitHubApiConstants.BASE_URL +
```

Comment:
`GitHub` level - it is business level
this level shouldn't know anything about `url` , `endpoint` etc
move all this logic to `controller` level